From c60eb456034807071261f13c75e5b66fc9db63d4 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Thu, 30 Nov 2017 14:22:45 +0800
Subject: [PATCH 2942/5242] MLK-17042 ARM64: dts: imx8mq: add apb clock for
 irqsteer

commit  d7c06df004c040439b065f0acaf09f0425217a1b from
https://source.codeaurora.org/external/imx/linux-imx.git

Add the 'IMX8MQ_CLK_DISP_APB_ROOT' clock to irqsteer
'clocks' property.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Reviewed-by: Sandor Yu <sandor.yu@nxp.com>
Reported-by: Anson Huang <anson.huang@nxp.com>
Reported-by: Pandy Gao <pandy.gao@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index fdb0b67..6664d2e 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -455,7 +455,7 @@
 		#interrupt-cells = <2>;
 		nxp,irqsteer_chans = <2>;
 		nxp,endian = <1>;		/* MSB */
-		clocks = <&clk IMX8MQ_CLK_DUMMY>;
+		clocks = <&clk IMX8MQ_CLK_DISP_APB_ROOT>;
 		clock-names = "ipg";
 	};
 
-- 
1.7.9.5

